### PR TITLE
add `Updating the URL for a repository` section to our contribution guidelines

### PR DESCRIPTION
**Pull request by ericphanson**
To provide guidance on helping with such PRs. I also tweaked the formatting to facilitate adding another section.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -52,13 +53,28 @@ Any comment without `[noblock]` included in it will block AutoMerge from automat
...
+This is done by manually making a PR to General to update the URL.
+You can review such a PR by checking that the old URL redirects to the new one.
+* If it does, that's a clear sign that the change is legitimate and the new URL is correct.
+  If you have write permissions to General, you can merge the PR; otherwise you can approve it or comment.
+* If it does not, you can ask the author why. This should be handled on a case-by-case basis.
+
+### Other ways to help
 
-Besides helping out with new package registrations directly, there are other ways to help out as well.
+Besides helping out with PRs to General directly, you can...
```
Comment (ericphanson) [inline CONTRIBUTING.md:72]:
```suggestion
Besides helping out with PRs to General, you can...
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -15,9 +15,10 @@ The [FAQ](FAQ) helps answer many more questions, like [do I need to register a p
 
 You (yes, you!) can help General be the best registry it can be.
 
-The first step is to check out new package registrations.
-They are filed under the ["new package" label](https://github.com/JuliaRegistries/General/pulls?q=is%3Apr+is%3Aopen+label%3A%22new+package%22), and a automatic feed posts them in the `#new-packages-feed` channel in the [community Slack](https://julialang.org/slack/) or [Zulip](https://julialang.zulipchat.com/register/).
+### New package registrations
 
+The first step is to getting involved with General is to check out new package registrations.
```
Comment (giordano) [inline CONTRIBUTING.md:20]: I think it should be
```suggestion
The first step to getting involved with General is to check out new package registrations.
```
?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -52,13 +53,28 @@ Any comment without `[noblock]` included in it will block AutoMerge from automat
...
+The most common reason is to update the URL for a repository.
+
+#### Updating the URL for a repository
+
+If someone transfers a GitHub repository, [we ask](https://github.com/JuliaRegistries/General#how-do-i-transfer-a-package-to-an-organization-or-another-user) that they update the URL stored in General.
+This is done by manually making a PR to General to update the URL.
+You can review such a PR by checking that the old URL redirects to the new one.
+* If it does, that's a clear sign that the change is legitimate and the new URL is correct.
+  If you have write permissions to General, you can merge the PR; otherwise you can approve it or comment.
+* If it does not, you can ask the author why. This should be handled on a case-by-case basis.
```
Comment (giordano) [inline CONTRIBUTING.md:68]: Explain that this requires to make sure all registered revisions (that is, their git tree sha1s) are accessible?

Comment (ericphanson) [reply CONTRIBUTING.md:68]: We could probably automate that right? Could run a workflow on every non-automated PR, check if a url changed, and if so check if all the versions exist.

Comment (ericphanson) [reply CONTRIBUTING.md:68]: But yea for this PR I should at least mention it

Comment (giordano) [reply CONTRIBUTING.md:68]: If that's doable, that'd be helpful!

Comment (ericphanson) [reply CONTRIBUTING.md]: ok, I mentioned this + checking that the change is legit

**Review by giordano — APPROVED** [review]

**Pull request merged**